src/tools/splash2txt/splash2txt.cpp

```diff
@@ -159,8 +159,9 @@ bool parseOptions( int argc, char** argv, ProgramOptions &options )
         }
 
     }
-    catch ( boost::program_options::error e )
+    catch ( const boost::program_options::error& )
     {
+        errorStream << desc << "\n";
```

Comment:
oh I forgot that I added that. it's just printing "--help" before throwing the error that something in the input was parsed wrong.